### ISSUE TITLE
Coq 8.15

### ISSUE THREAD

**Issue by silene** [EugeneLoy/coq_jupyter#45]
It seems like the protocol changed in Coq 8.15. I add to modify `ADD_COMMAND_TEMPLATE` as follows (and `_build_add_command` accordingly).
```xml
<call val="Add">
  <pair>
    <pair>
      <pair>
        <pair>
          <string></string>
          <int>0</int>
        </pair>
        <pair>
          <state_id val=""/>
          <bool val="false"/>
        </pair>
      </pair>
      <int>0</int>
    </pair>
    <pair>
      <int>0</int>
      <int>0</int>
    </pair>
  </pair>
</call>
```

I could submit it as a pull request, depending on what your stance is on supporting multiple versions of the protocol.

**Comment by EugeneLoy** [comment]
Hi. Thanks for reporting this.

> I could submit it as a pull request, depending on what your stance is on supporting multiple versions of the protocol.

**Comment by EugeneLoy** [comment]
Sorry for delay. This is now implemented and released as part of v1.6.0
